exchange/bidder_test.go

```diff
@@ -146,7 +147,8 @@ func TestMultiBidder(t *testing.T) {
 	}
 	bidder := adaptBidder(bidderImpl, server.Client())
 	currencyConverter := currencies.NewRateConverterDefault()
-	seatBid, errs := bidder.requestBid(context.Background(), &openrtb.BidRequest{}, "test", 1.0, currencyConverter.Rates())
+	reqInfo := adapters.ExtraRequestInfo{}
+	seatBid, errs := bidder.requestBid(context.Background(), &openrtb.BidRequest{}, "test", 1.0, currencyConverter.Rates(), reqInfo)
 
```

Comment:
Same here. If we really need the `ExtraRequestInfo` structure to exist we can get rid of the instantiation of `reqInfo` in line 150 and just pass `adapters.ExtraRquestInfo{}` as the last parameter just as they passed  `&openrtb.BidRequest{}` as second parameter